doctr/models/recognition/sar/pytorch.py

```diff
@@ -184,7 +185,7 @@ def __init__(
         vocab: str,
         rnn_units: int = 512,
         embedding_units: int = 512,
-        attention_units: int = 64,
+        attention_units: int = 512,
```

Comment:
Could you comment on this change?
Is this different from the paper?

Reply:
this 64 units was a mistake from my side i have had in mind 1x1x64 while computing the glimpse but it was 1x1xd and hidden_dim is in the complete model specified with 512 units :)